src/options.c

```diff
@@ -72,9 +72,9 @@ int options_parse_args(struct Options *opt, int argc, char **argv) {
       opt->logfile = optarg;
       break;
     case 'v': // verbose
```

Comment:
This is supposed to be applyable multiple times. This will break that functionality.